chew_test.go

```diff
@@ -148,3 +151,108 @@ func Test_getProcessor(t *testing.T) {
 		})
 	}
 }
+
+func TestProcess(t *testing.T) {
+	// today I learned that httptest.NewServer is a thing
```

Comment:
This is a text you could put in the commit message.

Here it will look strange when someone will read it later